### PR TITLE
[zh] localize blog:2022-08-29-csi-inline-volumes-ga.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh] localize blog:2022-08-29-csi-inline-volumes-ga.md

### CODE REVIEW

**Comment by Michelle951** [comment]
It seems that this is duplicate with https://github.com/kubernetes/website/pull/36806, which is earlier than this one.

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+<!--
+## Security Considerations
+
+Special consideration should be given to which CSI drivers may be used as inline volumes. `volumeAttributes` are typically controlled through the `StorageClass`, and may contain attributes that should remain restricted to the cluster administrator. Allowing a CSI driver to be used for inline ephmeral volumes means that any user with permission to create pods may also provide `volumeAttributes` to the driver through a pod spec.
+
+Cluster administrators may choose to omit (or remove) `Ephemeral` from `volumeLifecycleModes` in the CSIDriver spec to prevent the driver from being used as an inline ephemeral volume, or use an [admission webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/) to restrict how the driver is used.
+-->
+## 安全注意事项
+
+应特别考虑哪些 CSI 驱动程序可用作内联卷。`volumeAttributes` 通常是通过 `StorageClass` 控制的，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:158]:
```suggestion
你需要特别注意哪些 CSI 驱动可用作内联卷。`volumeAttributes` 通常是通过 `StorageClass` 控制的，
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+并且可能包含应该仍然仅限于集群管理员的属性。允许将 CSI 驱动程序用于内联临时卷意味着任何有权创建
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:159]:
```suggestion
并且可能包含仍应仅限于集群管理员的属性。允许将 CSI 驱动用于内联临时卷意味着任何有权创建
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+Pod 的用户也可以通过 Pod 规范向驱动程序提供 "volumeAttributes"。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:160]:
```suggestion
Pod 的用户也可以通过 Pod 规约向驱动提供 `volumeAttributes`。
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+- [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volumes)
+- [CSI documentation](https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html)
+- [KEP-596](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/596-csi-inline-volumes/README.md)
+- [Beta blog post for CSI Inline Volumes](https://kubernetes.io/blog/2020/01/21/csi-ephemeral-inline-volumes/)
+-->
+## 参考
+
+有关此功能的更多信息，请参阅：
+
+- [Kubernetes 文档](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volumes)
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:176]:
```suggestion
- [Kubernetes 文档](/zh-cn/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volumes)
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+- The CSI driver requires `volumeAttributes` that should be restricted to an administrator
+-->
+## 何时使用此功能
+
+CSI 内联卷适用于应该遵循 Pod 生命周期的简单本地卷。 
+它们对于从 CSI 驱动程序向 Pod 提供机密、配置数据或其他专用存储可能很有用。
+
+在以下情况下，CSI 驱动程序不适合内联使用：
+
+- 卷需要比 Pod 的生命周期更长的时间
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:67]:
```suggestion
- 卷需要持续的时间比 Pod 的生命周期长
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+- 需要卷快照、克隆或卷扩展
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:68]:
```suggestion
- 卷需要快照、克隆或扩容
```

File: content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+- CSI 驱动程序需要应仅限于管理员的 "volumeAttributes"
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md:69]:
```suggestion
- CSI 驱动程序需要应受限于管理员的 `volumeAttributes`
```

**Review by windsonsea — COMMENTED** [review]
It will be better to keep this word consistent in this topic:

Feature 特性

**Pull request closed without merging**